README.rst

```diff
@@ -63,6 +63,12 @@ PostgreSQL instance, (make sure you installed `podman-compose
   $ # Also available from PyPI: pip install podman-compose
   $ podman-compose up
 
+Then, sync your local instance in your local ``pgsql``:
```

Comment:
do we want to have a section with more general steps referencing other specific sections of the readme? wdyt?

pre-requisites: 
- make sure you have podman-compose installed
- make sure you are in an environment created with pipenv install --dev

the procedure for developing locally the first time is:

1. have a pg dump (retrieve from aws s3)
2. run podman-compose up to scale up pods for database and pgweb
3. run command to sync the pg dump into the local database (psql -h ...)
4. now you are ready to test new queries or create new migrations (Generating migrations and schema adjustment in deployment` section)

if you already have a database locally that you created in the past, make sure it is not too out of date and rember to follow `Generating migrations and schema adjustment in deployment` section

Reply:
Otherwise the comment you added looks good to me!

Reply:
BTW these instructions should be already documented in the "Automatic backups of Thoth deployment" section.

Reply:
Looks good to me, thanks for the suggestion :+1:

Reply:
> BTW these instructions should be already documented in the "Automatic backups of Thoth deployment" section.

Yes but it seems they describe each some specific action, adding something that put them all togheter on specific scenario could be useful, wdyt @fridex ?

Reply:
I think having this kind of introduction in a separate section could be useful as a kind of summary with complete instructions to develop locally, even if this would be a bit redundant